src/Microsoft.ML.Mkl.Redist/Microsoft.ML.Mkl.Redist.csproj

```diff
@@ -10,20 +10,32 @@
     <PackageTags>$(PackageTags) MLNET MKL</PackageTags>
     <TargetsForTfmSpecificBuildOutput>$(TargetsForTfmSpecificBuildOutput);CopyProjectReferencesToPackage</TargetsForTfmSpecificBuildOutput>
   </PropertyGroup>
+
   <PropertyGroup>
     <IncludeMLNetNotices>false</IncludeMLNetNotices>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
```

Comment:
This will set the include build output false for both the outer build and inner build, set it to false just for inner builds